tests/dtypes/test_array.py

```diff
@@ -0,0 +1,181 @@
+    app.post('/example/lvl3/Country', json={
+        '_id': lietuva_id,
+        'name': 'Lietuva',
+        'cities': [
+            {'name': 'Vilnius'},
+            {'name': 'Kaunas'}
+        ]
+    })
+
+    result = app.get('/example/lvl3/Country')
+    assert result.json()['_data'][0]['cities'] == []
+
+    result = app.get('/example/lvl3/Country?expand()')
+    assert result.json()['_data'][0]['cities'] == [
+        {'name': 'Vilnius'},
+        {'name': 'Kaunas'},
+    ]
+
+    result = app.get(f'/example/lvl3/Country/{lietuva_id}')
+    assert result.json()['cities'] == [
+        {'name': 'Vilnius'},
+        {'name': 'Kaunas'},
+    ]
+
+
+def test_getall_simple_type(rc, postgresql: str, request: FixtureRequest):
+    context = bootstrap_manifest(rc, '''
+    d | r | b | m | property | type    | ref     | access | level
+    example/simple           |         |         |        |
+      |   |   | Country      |         |         |        |
+      |   |   |   | name     | string  |         | open   |
+      |   |   |   | cities[] | string  |         | open   |
+    ''', backend=postgresql, request=request)
+    app = create_test_client(context)
+    app.authorize(['spinta_set_meta_fields', 'spinta_getone'])
+
+    lietuva_id = '6ffebdd7-2c5f-4e1f-b38b-e7a5c35e7216'
+    app.authmodel('example/simple/Country', ['insert', 'getall', 'search'])
+    app.post('/example/simple/Country', json={
+        '_id': lietuva_id,
+        'name': 'Lietuva',
+        'cities': [
+            'Vilnius',
+            'Kaunas'
+        ]
+    })
+
+    result = app.get(f'/example/simple/Country/{lietuva_id}')
+    assert result.json()['cities'] == [
+        'Vilnius',
+        'Kaunas'
+    ]
+
+
+def test_array_auth(rc, postgresql: str, request: FixtureRequest):
+    context = bootstrap_manifest(rc, '''
+    d | r | b | m | property    | type    | ref      | access
+    example/dtypes/array        |         |          |
+                                |         |          |
+      |   |   | Language        |         |          |
+      |   |   |   | name        | string  |          | open
+                                |         |          |
+      |   |   | Country         |         |          |
+      |   |   |   | name        | string  |          | open
+      |   |   |   | languages[] | ref     | Language | open
+    ''', backend=postgresql, request=request)
+    app = create_test_client(context)
+    app.authorize(['spinta_set_meta_fields'])
+    app.authmodel('example/dtypes/array', ['insert', 'getone', 'getall'])
+
+    LIT = "c8e4cd60-0b15-4b23-a691-09cdf2ebd9c0"
+    app.post('example/dtypes/array/Language', json={
+        '_id': LIT,
+        'name': 'Lithuanian',
+    })
+
+    LT = 'd73306fb-4ee5-483d-9bad-d86f98e1869c'
+    app.post('example/dtypes/array/Country', json={
+        '_id': LT,
+        'name': 'Lithuania',
+        'languages': [
+            {'_id': LIT}
+        ]
+    })
+
+    app.unauthorize()
```

Comment:
To reproduce issue with https://github.com/atviriduomenys/spinta/pull/356/commits/817ccd4211d943b6e409d2f05b16003744d7881d, we need to drop autorization and then proceed with unauthorized client.

Reply:
issue was, that tabular array shortcut, didn't inherit access, because of it, all of the arrays created with shortcut were private, changed tests, to see if inheritance works